src/order/locally_finite.lean

```diff
@@ -313,6 +313,9 @@ def Iio (b : α) : multiset α := (finset.Iio b).val
 end order_bot
 end multiset
 
+@[simp] lemma finset.Ico_val [preorder α] [locally_finite_order α] (a b : α) :
```

Comment:
This lemma is already the equational lemma for `multiset.Ico`. So you don't need it.